packages/dds/sequence/src/test/fuzz/sharedString.fuzz.spec.ts

```diff
@@ -103,3 +103,28 @@ describe("SharedString fuzz testing with rebased batches", () => {
 		// replay: 0,
 	});
 });
+
+// todo: potentially related to AB#7050
```

Comment:
Could you mention briefly that a few seeds are minimized and written out in intervalRebasing.spec.ts? Also probably good to write something similar near those tests